app/workers/articles/update_page_views_worker.rb

```diff
@@ -2,16 +2,18 @@ module Articles
   class UpdatePageViewsWorker
     include Sidekiq::Worker
 
-    sidekiq_options queue: :medium_priority,
+    sidekiq_options(queue: :medium_priority,
```

Comment:
I added parenthesis around this as my buffer parser was insisting on some odd indentation for documentation.